docs/developing_recipes_locally.md

```diff
@@ -0,0 +1,207 @@
+
+```sh
+conan config install
+```
+
+## Basic Commands
+
+We recommend working from the `recipes/project` folder itself. You can learn about the [recipe file structure](how_to_add_packages.md#recipe-files-structure) to understand the folder and files located there.
+
+> **Note**: You can only change one recipe per pull request, and working from the [_recipe folder_](hhow_to_add_packages.md#the-recipe-folder-conanfilepy) will help prevent making a few mistakes. The default for this folder is `all`, follow the link above to learn more.
+
+The [entire workflow of a recipe](https://docs.conan.io/en/latest/developing_packages/package_dev_flow.html) can be execute with the [`conan create`](https://docs.conan.io/en/latest/reference/commands/creator/create.html). This should look like:
+
+* `conan create all/conanfile.py 0.0.0@ -pr:b=default -pr:h=default`
+
+ConanCenter also has a few [support settings/options](supported_platforms_and_configurations.md) which highly recommend to test. For example
+`conan create all/conanfile.py 0.0.0@ -o project:shared=True -s build_type=Debug` is a easy way to more sure the package is correct.
+
+### Try it yourself
+
+For instance you can create packages for `fmt` in various supported configurations by running:
+
+```sh
+cd recipes/fmt
+conan create all/conanfile.py fmt/9.0.0@ -pr:b=default -pr:h=default
+conan create all/conanfile.py fmt/9.0.0@ -o fmt:header_only=True -pr:b=default -pr:h=default
+conan create all/conanfile.py fmt/9.0.0@ -s build_type=Debug -o fmt:shared=True -pr:b=default -pr:h=default
+```
+
+## Debugging Failed Builds
+
+Some common errors related to Conan can be found on [troubleshooting](https://docs.conan.io/en/latest/faq/troubleshooting.html) section.
+For ConanCenter Hook errors, go to the [Error Knowledge Base](error_knowledge_base.md) page to know more about those.
+
+To test with the same environment, the [build images](supported_platforms_and_configurations.md#build-images) are available.
+Instructions for using these images can be found in [Testing more environments](#testing-more-environments)
+
+## Running the Python Linters
+
+Linters are always executed by Github actions to validate parts of your recipe, for instance, if it uses migrated Conan tools imports.
+To understand how to run linters locally, read [V2 linter](v2_linter.md) documentation.
+
+It is possible to run the linter locally the same way it is being run [using Github actions](../.github/workflows/linter-conan-v2.yml):
+
+ * (Recommended) Use a dedicated Python virtualenv.
+ * Ensure you have required tools installed: `conan` and `pylint` (better to uses fixed versions)
+
+   ```
+   pip install conan~=1.0 pylint==2.14
+   ```
+
+ * Set environment variable `PYTHONPATH` to the root of the repository
+
+   ```
+   export PYTHONPATH=your/path/conan-center-index
+   ```
+
+  * Now you just need to execute the `pylint` commands:
```

Comment:
It might be nice to leverage something like [pre-commit](https://pre-commit.com/) to make it easy to have the linters run before committing code locally. I'm liable to forget.

Reply:
Me too 🙊 one day we will have an easier way than in the PR

Reply:
👏 

We should  add the spdx hook too